### ISSUE TITLE
Webmin missing perl dependency during install

### ISSUE THREAD

**Issue by Fffrank** [MichaIng/DietPi#1741]
#### Required Information:
- DietPi Version | 6.7
- SBC Device (EG: RPi 3)? Odroid C2
- Power supply used (EG: 5v 1A RAVpower)? 2.5A Amazon
- SD card used (EG: Sandisk ultra)? Sandisk Ultra
- Distro (EG: Jessie) | Jessie

#### Additional Information (if applicable):
- Software title? WebMin
- Can this issue be replicated on a fresh installation of DietPi? Yes.

#### Expected behaviour:
WebMin installs through dietpi-software

#### Actual behaviour:
Webmin Fails due to missing dependency (perl)

#### Steps to reproduce:
Install WebMin on fresh Dietpi

#### Did you submit a ```dietpi-bugreport```?
No.

**Comment by MichaIng** [comment]
@Fffrank 
Thanks for your report.

Even worse on my Stretch VM:
```
dpkg: dependency problems prevent configuration of webmin:
 webmin depends on libnet-ssleay-perl; however:
  Package libnet-ssleay-perl is not installed.
 webmin depends on libauthen-pam-perl; however:
  Package libauthen-pam-perl is not installed.
 webmin depends on libio-pty-perl; however:
  Package libio-pty-perl is not installed.
 webmin depends on apt-show-versions; however:
  Package apt-show-versions is not installed.
```

We should use their dependency list: http://webmin.com/deb.html
`apt-get install perl libnet-ssleay-perl openssl libauthen-pam-perl libpam-runtime libio-pty-perl apt-show-versions python`

**€: But ah wait, this is expected behaviour, we let APT run into dependency issue and then let `apt-get -f install` automatically install/fix them. This way we don't need to maintain a dependency list, that can change:**
```
dpkg: error processing package webmin (--install):
 dependency problems - leaving unconfigured
Processing triggers for systemd (232-25+deb9u3) ...
Errors were encountered while processing:
 webmin
[  OK  ] DietPi-Software | APT fix, please wait...
Reading package lists...
Building dependency tree...
Reading state information...
Correcting dependencies... Done
The following additional packages will be installed:
  apt-show-versions libapt-pkg-perl libauthen-pam-perl libio-pty-perl
  libnet-ssleay-perl perl-openssl-defaults
The following NEW packages will be installed:
  apt-show-versions libapt-pkg-perl libauthen-pam-perl libio-pty-perl
  libnet-ssleay-perl perl-openssl-defaults
0 upgraded, 6 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
```

Can you verify that finally `perl` got installed and `webmin` successfully afterwards?

**Comment by luispuerto** [comment]
I think I have the same issue and I summited a bug report: 
c1086a6e-a26a-4ef4-ad97-44ebd15b46c7-0

I've just 
```sh
apt-get install perl libnet-ssleay-perl openssl libauthen-pam-perl libpam-runtime libio-pty-perl apt-show-versions python
```
And now it works.

**Comment by Fourdee** [comment]
@luisspuerto 

Thanks for the report 👍 

I'll move this to v6.9 and investigate this further.

**Comment by Fourdee** [comment]
Ok, perl/python are required at the very least for G_AGF to resolve. We could manually add the required pre-reqs to install list, however, could change on distro version and .deb update of webmin.

--- 

Install initially fixed, ~page does not load, endless wait.~ httpS required. Will make that clearer with online doc.

**Comment by Fourdee** [comment]
Resolved for v6.9.
